todo-app/src/App.js

```diff
@@ -1,10 +1,98 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import Container from './Components/Container'
+import { LoadFromLocalStorage, SaveToLocalStorage} from './localStorage'
+
+import './App.css';
+import data from './todoList.json';
 
 class App extends Component {
+  state = {
+    todoItems: LoadFromLocalStorage() || data
+  }
+
+  componentDidUpdate = () => {
```

Comment:
here i have removed the lifecycle methods  and used the imported file "localSrotrage.js" and i have changed the state